pyvista/core/pointset.py

```diff
@@ -120,6 +121,96 @@ def remove_cells(self, ind, inplace=True):
         target.RemoveGhostCells()
         return target
 
+    def points_to_double(self, inplace=False):
+        """Convert the points datatype to double precision.
+
+        Examples
+        --------
+        Create a mesh that has points of the type ``float32`` and
+        convert the points to ``float64``.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh.points.dtype
+        dtype('float32')
+        >>> mesh.points_to_double()
+        >>> mesh.points.dtype
+        dtype('float64')
```

Comment:
Doesn't this example only work if `inplace=True` is the default?

Reply:
`points_to_double` is an interesting case... I'm leaning towards letting it be an exception to the rule where it perhaps only operates inplace

In any case, I have yet to go through the examples/tests where these methods are used to make sure they comply with the changes.

Reply:
I’d prefer leaving this as an inplace only method, or at least default to inplace.